operator/pkg/common/extensions/reconcile_scanner_v4_db_password.go

```diff
@@ -0,0 +1,66 @@
+package extensions
```

Comment:
Have you considered slightly generalizing the existing reconciler code for scanner v2 to cover the v4 case as well? Or do you see good reasons for not doing it here?